homeassistant/components/islamic_prayer_times/strings.json

```diff
@@ -8,7 +8,7 @@
       }
     },
     "abort": {
-      "one_instance_allowed": "Only a single instance is necessary."
+      "one_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]"
```

Comment:
It's good to also update the key and use a standard one, and then update the config flow and tests too that use the key.

Reply:
@MartinHjelmare Sorry, I don't entirely understand what you mean by using a standard key. I've copied that string from another integration referencing to [these docs](https://developers.home-assistant.io/docs/internationalization/core/#introducing-new-strings). Still trying to better understand the architecture of HA core as I am new.

Thanks for taking the time :)

Reply:
The standard key is usually the key that is used in the common strings.

https://github.com/home-assistant/core/blob/432133a12a705079c5013df12d0c387be5db80bc/homeassistant/strings.json#L50

There's one exception to this rule: `already_configured` is the standard key for that abort reason. We have more fine grained reasons in the common strings that we reference when using that key.